### ISSUE TITLE
x/pkgsite:

### ISSUE THREAD

**Issue by taradaidv** [golang/go#66715]
Example URL: https://pkg.go.dev/404
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.4.1 Safari/605.1.15

Hi, the placeholder image https://pkg.go.dev/static/shared/gopher/airplane-1200x945.svg has two problems in the dark mode. Since we always strive for perfection, let's not overlook the flaws in the brand image...

Bug:
<img width="945" alt="img_fix" src="https://github.com/golang/go/assets/35196630/f1a7e24f-447a-41fd-aed0-e53593af5617">

Fix:
<img width="945" alt="img_bug" src="https://github.com/golang/go/assets/35196630/d501144e-2902-4b91-8fe4-432a6b1680dd">

Attached a corrected image, with love for GO community:
![fix_airplane-1200x945](https://github.com/golang/go/assets/35196630/5d82910f-f52c-4a67-a554-e3ded631a079)

**Comment by adonovan** [comment]
You have a keen eye! I'll have our team graphic designers get on it at once. ;-)

**Comment by linsite** [comment]
👍 I believe it's fixed by now. Please check it. @taradaidv

**Comment by taradaidv** [comment]
> 👍 I believe it's fixed by now. Please check it. [@taradaidv](https://github.com/taradaidv)

Cleared the cache in the browser before checking.
Checked, no changes, white spots still there. Maybe the CDN cache hasn't updated.

**Comment by linsite** [comment]
Yes, you are right. My browser's background is white, failed to see that, my bad.
